contracts/RelayHub.sol

```diff
@@ -237,8 +238,8 @@ contract RelayHub is IRelayHub {
         // We've already checked that the paymaster has enough balance to pay for the relayed transaction, this is only
         // a sanity check to prevent overflows in case of bugs.
         require(balances[relayRequest.relayData.paymaster] >= charge, "Should not get here");
```

Comment:
your safemath actually make this strange "require" line useless:

Reply:
Should I remove this line?